packages/devextreme/js/__internal/grids/grid_core/views/m_columns_view.ts

```diff
@@ -1376,7 +1376,7 @@ export class ColumnsView extends ColumnStateMixin(modules.View) {
     return columnIdentifier;
   }
 
-  public getColumnElements() {}
+  public getColumnElements(): any {}
```

Comment:
could you please provide accurate type instead of `any` type?

Reply:
> could you please provide accurate type instead of `any` type?

This will cause a lot of changes in the code. I think we need to do grid typing as a separate task.

Reply:
I'm ok to resolve the comment but could you please provide a bit of details about the changes which we need to do?